SE-XXX Typed throws.md

```diff
@@ -438,10 +467,16 @@ do {
     dump(error)
     recoverFromLibraryError(error)
 }
-
 ```
 
-Scenarios where we can make use of the Swift compiler with the current proposal:
+The most evident scenario is that no longer is needed to catch a certain type in the `catch` clauses, due to the fact that the generic `catch` clause will become strongly typed with the type specified in the function signature.
+
+This single simple scenario opens up to a whole family of possible scenarios that should be mentioned for the sake of clarity.
+In first place we will specify different examples where the Swift compiler interprets the new code and make decisions about the code in ways that it wasn't working before:
+
+### Inheritance
+
+A typed `throws` function cannot throw other type unrelated (by inheritance). If this happens, a compilation error will be generated as in the following example:

```

Comment:
I don't know where we have inheritance here?

Reply:
My point here was that, any typed throw where inheritance cannot be applied (like the example below) would throw an error. I'm just covering the negative scenario, maybe I should cover the positive one